shared/fs/index.js

```diff
@@ -64,29 +63,35 @@ const FileRow = RowConnector(({path, name, icon, onOpen}: FileRowProps) => (
   <ClickableBox onClick={onOpen} style={{...stylesCommonRow}}>
     <Box style={{...globalStyles.flexBoxRow, alignItems: 'center', flex: 1}}>
       <Icon type={icon} style={{marginRight: globalMargins.small}} />
-      <Box style={FolderBoxStyle}>
+      <Box style={folderBoxStyle}>
         <Text type="Body">{name}</Text>
       </Box>
     </Box>
   </ClickableBox>
 ))
 
-const Files = ({path, items, onBack}: FolderProps) => (
-  <Box style={styleOuterContainer}>
-    <Box style={stylesContainer}>
-      {onBack &&
-        Types.pathToString(path) !== '/keybase' && (
-          <Box style={globalStyles.flexBoxColumn}>
-            <BackButton onClick={onBack} style={{left: 16, position: 'absolute', top: 16}} />
-          </Box>
-        )}
-      <FolderHeader title={'Folders: ' + Types.pathToString(path)} />
-      <List
-        items={items}
-        renderItem={(index, item) => <FileRow key={Types.pathToString(item)} path={item} />}
-      />
-    </Box>
-  </Box>
-)
+class Files extends React.PureComponent<FolderProps> {
+  _renderRow = (index, item) => <FileRow key={Types.pathToString(item)} path={item} />
+
+  render() {
+    const onBack = this.props.onBack
+    const path = this.props.path
+    const items = this.props.items
+    return (
+      <Box style={styleOuterContainer}>
+        <Box style={stylesContainer}>
+          {onBack &&
+            Types.pathToString(path) !== '/keybase' && (
+              <Box style={globalStyles.flexBoxColumn}>
+                <BackButton onClick={onBack} style={{left: 16, position: 'absolute', top: 16}} />
```

Comment:
we likely don't want absolute positioned things like this. Also we have HeaderHoc for this kind of thing usually

Reply:
This is going away anyway since @cecileboucheron's design doesn't have back navigation in this flow.

Reply:
Also `HeaderHoc` uses `position: absolute`.